index.js

```diff
@@ -140,7 +146,21 @@ RingPop.prototype.setupChannel = function setupChannel() {
     createRingPopTChannel(this, this.channel);
 };
 
+RingPop.prototype.addLocalMember = function addLocalMember() {
+    this.membership.addMember({ address: this.hostPort });
+};
+
 RingPop.prototype.adminJoin = function adminJoin(target, callback) {
+    if (!this.membership.localMember) {
+        return callback(InvalidLocalMemberError());
```

Comment:
nitpick: For sanity a callback must always be EITHER sync or async. 

See http://blog.izs.me/post/59142742143/designing-apis-for-asynchrony

Recommend doing `process.nextTick(flushError)`